frontend/src/components/CreateMemo/components/Locations/SavedLocations.tsx

```diff
@@ -29,7 +34,7 @@ const SavedLocations = ({ id, reloadDropdown, onDropdownReloaded, onLocationSele
 
   useEffect(() => {
     fetchData();
-  }, [id, reloadDropdown, onDropdownReloaded]);
+  }, [currentUser.uid, props.reloadDropdown, props.onDropdownReloaded]);
```

Comment:
why watch on `currentUser.uid` in the useEffect subscriber array?

Reply:
oh I think before id was a useState that fetch from query so watch it to fetchData or something, just removed it in new commit